tms-service/src/app.ts

```diff
@@ -21,16 +22,16 @@ class App extends Koa {
   }
 
   configureMiddlewares(): void {
-    const readSwagger = swagger.loadDocumentSync('./mojaloop-api.yaml');
+    const readSwagger = swagger.loadDocumentSync(path.join(__dirname, 'mojaloop-api.yaml'));
```

Comment:
Why was this change needed?

Reply:
It was causing the import error after the build. Due to that, I converted to absolute import rather than a relative one.